_posts/0000-01-02-RicardoUValencia.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
It's awesome merges branch on GitHub 👍. If you want learn more follow me for more content 🤓 :)!!
:octocat: 
![steroidtocat](https://user-images.githubusercontent.com/87098001/174221273-989a0d6c-43ee-4dec-9ae6-6f590cdcdb4c.png)